openapi.yaml

```
@@ -1135,12 +1609,20 @@ components:
         enum:
           - true
           - false
+    DryRun:
+      in: query
+      name: dry-run
+      required: false
+      schema:
+        type: boolean
+        default: false
```

```suggestion
      schema:
        type: string
        default: false
        enum:
          - true
          - false
```

I now realise I've made this mistake in some of the other query parameters too, but I don't think query params work well with the `boolean` type, and it's better to use a string enum with just true/false.